Explorer/Assets/DCL/Multiplayer/Movement/Systems/RemotePlayersMovementSystem.cs

```diff
@@ -45,26 +43,18 @@ protected override void Update(float t)
             UpdateRemotePlayersMovementQuery(World, t);
         }
 
-        private static void HandleFirstMessage(FullMovementMessage firstRemote, ref CharacterTransform transComp, ref CharacterAnimationComponent anim, ref RemotePlayerMovementComponent remotePlayerMovement, in IAvatarView view)
+        private static void HandleFirstMessage(ref CharacterTransform transComp, NetworkMovementMessage firstRemote, ref RemotePlayerMovementComponent remotePlayerMovement)
```

Comment:
I recommend adding `in` modifier for large value types that you have no intention to modify as it will prevent copying the structure. It can be even visible in the Profiler when there are many entities